style.css

```diff
@@ -0,0 +1,244 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/*End CSS RESET */
+
+#wrapper {
+  position:relative;
+  left: 20%;
+  right: 15%;
+  width: 901px;
+	margin-top: 20px;
+}
+
+.custom-hr {
+  background-color: grey;
+  height:10px;
+  margin: 20px 0px;
+  background: repeating-linear-gradient(45deg,
+   #FFFFFF , #929292  1px, transparent 2px, transparent 3px);
+}
+
+.header{
+  display: inline-block;
+  width: 100%;
+}
+
+.header-left{
+  display: inline;
+  float: left;
+	width: 500px;
+}
+
+.header-left img{
+  display: inline;
+  float: left;
+  margin-right: 5px;
+}
+
+.cursive-text {
+  display: inline;
+	font-family: 'Yellowtail', cursive;
+	font-size: 35px;
+}
+
+.social-media img {
+  display:inline;
+  float: right;
+  margin:1px;
+}
+
+#rss {
+  margin-left: 8px;
+}
+
+#article {
+  display: inline-block;
+	margin-top: 40px;
+}
+/*
+#article img{
+  display: inline;
+  float:right;
+}
+*/
+.little-print{
+  display: inline;
+  float:right;
+}
+
+#article span{
+  display: inline;
+  float:right;
+  margin-left: 4px;
+}
+
+#article h1 {
+	font-family: 'Coda', cursive;
```

Comment:
keep your indentation consistent, this looks to be a different number of spaces than the rest of your css.